pkg/constants/constants.go

```diff
@@ -0,0 +1,16 @@
+package constants
+
+const (
+	SingletonName              = "instance"
+	OpenshiftNS                = "openshift-logging"
```

Comment:
not required in EO -- we should use whatever namespace our CRs are created in

Reply:
fixed